tests/python/distributed/test_collective_communication.py

```diff
@@ -386,6 +386,61 @@ def forward(self, x, y):
                 check(m_out, odd_out)
 
 
+@pytest.mark.skipif(skip_dist_test(min_rank_num=2, require_exact_rank=True), reason=SKIP_REASON)
+@pytest.mark.parametrize("computation", ["sum", "prod", "min", "max"])
+def test_reduce_scatter_input_tensor(computation):
```

Comment:
Make the name more informative.

```suggestion
def test_reduce_scatter_input_tensor_list(computation):
```